src/views/Home2/components/Hero.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
+import styled, { keyframes } from 'styled-components'
+import { Flex, Heading, Link, Button } from '@pancakeswap/uikit'
+import { useWeb3React } from '@web3-react/core'
+import { useTranslation } from 'contexts/Localization'
+import ConnectWalletButton from 'components/ConnectWalletButton'
+import useTheme from 'hooks/useTheme'
+import { SlideSvgDark, SlideSvgLight } from './SlideSvg'
+import CompositeImage, { getSrcSet, CompositeImageProps } from './CompositeImage'
+
+const flyingAnim = () => keyframes`
+  from {
+    transform: translate(0,  0px);
+  }
+  50% {
+    transform: translate(-10px, -15px);
+  }
+  to {
+    transform: translate(0, 0px);
+  }  
+`
+
+const fading = () => keyframes`
+  from {
+    opacity: 0.9;
+  }
+  50% {
+    opacity: 0.1;
+  }
+  to {
+    opacity: 0.9;
+  }  
+`
+
+const BgWrapper = styled.div`
+  z-index: -1;
+  overflow: hidden;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  bottom: 0px;
+  left: 0px;
+`
+
+const InnerWrapper = styled.div`
+  position: absolute;
+  width: 100%;
+  bottom: -3px;
+`
+
+const BunnyWrapper = styled.div`
+  width: 100%;
+  animation: ${flyingAnim} 2.5s ease-in-out infinite;
+`
+
+const StarsWrapper = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  & :nth-child(2) {
+    animation: ${fading} 2s ease-in-out infinite;
+    animation-delay: 1s;
+  }
+
+  & :nth-child(3) {
+    animation: ${fading} 5s ease-in-out infinite;
+    animation-delay: 0.66s;
+  }
+
+  & :nth-child(4) {
+    animation: ${fading} 2.5s ease-in-out infinite;
+    animation-delay: 0.33s;
+  }
+`
+
+const Hero = () => {
+  const { t } = useTranslation()
+  const { account } = useWeb3React()
+  const { theme } = useTheme()
+  const imagePath = '/images/home/lunar-bunny/'
```

Comment:
Define constants outside the component

Reply:
Nice, done- few places where I was doing this.